sdk/appcenter/src/main/java/com/microsoft/appcenter/AppCenter.java

```diff
@@ -180,6 +182,11 @@
      */
     private AppCenterHandler mAppCenterHandler;
 
+    /**
+     * The ingestion object used to send batches to the server.
+     */
+    private Ingestion mIngestion;
```

Comment:
remove that and replace by mLogUrl, see other comment.

Reply:
Resolve in new commit.